modules/tinymce/src/core/test/ts/webdriver/keyboard/SpaceKeyTest.ts

```diff
@@ -39,7 +43,7 @@ describe('webdriver.tinymce.core.keyboard.SpaceKeyTest', () => {
       if (isSafari) { // Split due to normalization issue. See TINY-8833
         TinyAssertions.assertContent(editor, '<p>s &nbsp;<span style="display: block;" contenteditable="false">a</span></p>');
       } else if (isFirefox) { // Split due to normalization issue. See TINY-8833
-        TinyAssertions.assertContent(editor, '<p>s&nbsp; <span style="display: block;" contenteditable="false">a</span></p>');
+        TinyAssertions.assertContent(editor, '<p>s&nbsp; <br><span style="display: block;" contenteditable="false">a</span></p>');
```

Comment:
This seems odd a `br` was added. What change caused us to have to update these test cases?

Reply:
@ArvinJ-H , if I remember correctly from the conversation with you, this only occurs on the most recent firefox. maybe adding a comment to explain why `br` is added here

Reply:
this test is skipped... waiting for these two ticket to be resolved
https://ephocks.atlassian.net/browse/TINY-10742
https://ephocks.atlassian.net/browse/TINY-10743